tomls/omnibus-arbitrum-sepolia/spot/dai.toml

```diff
@@ -0,0 +1,80 @@
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "getMarketOwner"
+fromCall.args = ["<%= extras.synth_dai_market_id %>"]
+func = "updatePriceData"
+args = [
+    "<%= extras.synth_dai_market_id %>",
+    "<%= extras.dai_oracle_id %>",
+    "<%= extras.dai_oracle_id %>",
+    "<%= settings.spotStrictStalenessTolerance %>",
+]
+
+[invoke.setSynthMaxMarketCollateralDai]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+fromCall.args = []
+func = "configureMaximumMarketCollateral"
+args = [
+    "<%= extras.synth_dai_market_id %>",
+    "<%= imports.dai_mock_collateral.contracts.MintableToken.address %>",
+    "<%= settings.synthDaiMaxMarketCollateral %>",
+]
+
+[invoke.configureSynthWrapperDai]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "getMarketOwner"
+fromCall.args = ["<%= extras.synth_dai_market_id %>"]
+func = "setWrapper"
+args = [
+    "<%= extras.synth_dai_market_id %>",
+    "<%= imports.dai_mock_collateral.contracts.MintableToken.address %>",
+    "<%= settings.synthDaiMaxMarketCollateral %>",
+]
+
+[invoke.addSynthSettlementStrategyDai]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "getMarketOwner"
+fromCall.args = ["<%= extras.synth_dai_market_id %>"]
+func = "addSettlementStrategy"
+args = [
+    "<%= extras.synth_dai_market_id %>",
+    { strategyType = "1", settlementDelay = "0", settlementWindowDuration = "1", priceVerificationContract = "0x0000000000000000000000000000000000000000", feedId = "0x0000000000000000000000000000000000000000000000000000000000000000", url = "", settlementReward = 0, minimumUsdExchangeAmount = "0", maxRoundingLoss = "0", priceDeviationTolerance = "0", disabled = false },
+]
+extra.synth_dai_settlement_strategy_id.event = "SettlementStrategyAdded"
+extra.synth_dai_settlement_strategy_id.arg = 1
+
+[invoke.setSynthSettlementStrategyDai]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "getMarketOwner"
+fromCall.args = ["<%= extras.synth_dai_market_id %>"]
+func = "setSettlementStrategy"
+args = [
+    "<%= extras.synth_dai_market_id %>",
+    "<%= extras.synth_dai_settlement_strategy_id %>",
+    { strategyType = "1", settlementDelay = "<%= settings.bigCapSettlementDelay %>", settlementWindowDuration = "<%= settings.bigCapSettlementWindowDuration %>", priceVerificationContract = "<%= settings.pyth_price_verification_address %>", feedId = "<%= settings.pythDaiFeedId %>", url = "<%= settings.pythFeedUrl %>", settlementReward = 0, minimumUsdExchangeAmount = "<%= settings.settlementMinimumUsdExchangeAmount %>", maxRoundingLoss = "<%= settings.settlementMaxRoundingLoss %>", priceDeviationTolerance = "<%= settings.priceDeviationTolerance %>", disabled = false },
+]
+
+[invoke.setSynthAsyncFixedFeeDai]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "getMarketOwner"
+fromCall.args = ["<%= extras.synth_dai_market_id %>"]
+func = "setAsyncFixedFee"
+args = ["<%= extras.synth_dai_market_id %>", "<%= settings.synthDaiAsyncFixedFee %>"]
```

Comment:
NOTE that we do **NOT** need to set `depends` for these steps because there is no more `parseEther`. It is moved to much better suited place: 
```toml
[setting.synthDaiAsyncFixedFee]
defaultValue = "<%= parseEther('0.1') %>"
```